README.md

```diff
@@ -1 +1,137 @@
-# CorrelationsRecorderTemplates
\ No newline at end of file
+<br>
+
+<h4 align="center">Correlations Recorder Templates</h4>
+
+<p align="center">
+  <a href="#about">About</a> •
+  <a href="#prerequisites">Prerequisites</a> •
+  <a href="#creating-templates">Creating Templates</a> •
+  <a href="#sharing-templates">Sharing Templates</a>
+</p>
+
+# About
+
+Correlations Recorder Central Repository its a community where users can share their Correlation Templates to be used, and automatically downloaded into the [Correlation's Recorder JMeter Plugin](https://github.com/Blazemeter/CorrelationRecorder).
+
+# Prerequisites
+
+In order to download, update and share templates you need to:
+
+1. Download and Install JMeter
+1. Install JMeter Plugin Manager
+1. Install Correlations Recorder Plugin
+
+# Creating Templates
+
+From there, you can either start creating your templates from scratch or, load and edit one from the Central Repository, loaded by default into your machine.
+
+## Building from Scratch
+
+Here you can use the Template we prepare for you or, just add the Component as it is. 
+
+### Using the Template
+
+1. Go to JMeter Templates
+1. Select "Correlation Recorder"
+1. Select the "Correlation Recorder" component
+
+### Adding the Component
+
+1. Click in your Test Plan
+1. Follow this route: Add > Non-Test Element > "Correlation Controller"
+1. Select the "Correlation Recorder" Component
+
+---
+
+From this point onwards, its just about adding Rules by clicking the "Add Button" and, when you are ready, save your Correlation Template.
+
+## Using one from This Repository
```

Comment:
What do you think about `Using a template from Central Repository`? Just to have more context from outside of this repository. Remember that we will reference this document, more precisely this section many times!